webpack.config.js

```diff
@@ -3,18 +3,19 @@
 var webpack = require('webpack');
 
 module.exports = {
-    entry: './src/timekit.js',
-    devtool: 'source-map',
-    output: {
-        path: './dist',
-        filename: 'timekit-sdk.js',
-        libraryTarget: 'umd',
-        library: 'timekit'
-    },
-    plugins: [
-        new webpack.IgnorePlugin(/vertx/),
-        new webpack.ProvidePlugin({
-          Promise: 'es6-promise-promise'
-        })
-    ]
+	mode: 'development',
+	entry: './src/timekit.js',
+	devtool: 'source-map',
+	output: {
+		path: __dirname + '/dist',
```

Comment:
Any reason we changed this?

Reply:
This change is a part of webpack upgrade from 1 to 4